modules/op-configuring-buildah-to-use-build-user.adoc

```diff
@@ -10,11 +10,11 @@ You can define a Buildah task to use the `build` user with user id `1000`.
 
 .Procedure
 
-. Create a copy of the `buildah` cluster task as an ordinary task.
+. Create a copy of the `buildah` task, which is provided in the `openshift-pipelines` namespace, modified to be an ordinary task.
 +
 [source,terminal]
 ----
-$ oc get clustertask buildah -o yaml | yq '. |= (del .metadata |= with_entries(select(.key == "name" )))' | yq '.kind="Task"' | yq '.metadata.name="buildah-as-user"' | oc create -f -
+$ oc get -n openshift-pipelines task buildah -o yaml | yq '. |= (del .metadata |= with_entries(select(.key == "name" )))' | yq '.kind="Task"' | yq '.metadata.name="buildah-as-user"' | oc create -f -
```

Comment:
While it's very subjective and produces the same result, I would reorder argument so that the command reads more naturally
```
oc get task buildah -n openshift-pipelines -o yaml ...
```

Reply:
fixing but also is the change of Kind to be removed now?